Farmacy/ViewController/LoginViewController.m

```diff
@@ -30,6 +32,49 @@ - (void)viewDidLoad {
     
 }
 
+//For who don't want to login with Facebook account
+- (IBAction)loginUserWithParse:(id)sender {
+    NSString *username = self.usernameTextField.text;
+    NSString *password = self.passwordTextField.text;
```

Comment:
It's not a good idea to compare/save/display the user the password. You can encrypt it, some options https://en.wikipedia.org/wiki/Secure_Hash_Algorithms. But for intern project, you can keep it like this to unblock other features implementation. Please keep this as a long term action item. Thanks.